### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN015.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#61] [General]
# https://blog.3place.cn/3place/lmyn/LMYN015.html

https://blog.3place.cn/3place/lmyn/LMYN015.html

**Comment by picklecai** [comment]
罗波那出场了。原来大boss都是得到神的眷顾的。

**Comment by picklecai** [comment]
这些神灵聚集在一起，
按照仪式坐在座位上；
他们对着宇宙的创造者
那大梵天都把话来讲

神，一边吃，一边开小会。

**Comment by picklecai** [comment]
由于他看不起凡人，这罗刹没有提凡人的名。因此，只有凡人能杀死他，任何别的办法都不行。

一个漏洞。
都要懂枚举才好。

**Comment by picklecai** [comment]
有点像《杨乃武与小白菜》：底层人之所以告状成功，是因为上层的人有自己的问题要解决。
十车王之所以求子成功，只是因为神仙们有自己的仇怨要解决。

**Comment by picklecai** [comment]
这个坏蛋威胁天地三界，
他与一切高贵的人为敌，
他甚至想伤害天帝释〔178〕，
那位三十三天的大帝。 （1.4.8）
仙人、夜叉〔179〕、乾闼婆，
再加上阿修罗和婆罗门，
这一个坏蛋都去攻击，
您的恩宠使他发了昏。 

阿修罗，在这里不是坏人。

**Comment by picklecai** [comment]
天神们对毗湿奴说的话，分几个层次：
第一层：为百姓造福。
第二层：十车王夫妇都很优秀。
第三层：托生。
第四层：任务。
第五层：任务对象的特点和恶行。

在说到第三层之前，毗湿奴应该都是懵圈的：我怎么就能造福了？这几个凡人和我有啥关系？
到最后一层，才说这个任务的困难点。

这帮神仙，鸡不鸡贼呀！
